src/Tizen.NUI/src/public/Window.cs

```diff
@@ -1272,6 +1272,24 @@ internal void SetPositionSize(Rectangle positionSize)
             // Setting Position of the window should request a relayout of the tree.
         }
 
+        /// <summary>
+        /// Add FrameCallback
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void AddFrameCallback(FrameCallbackInterface frameCallback)
+        {
+            frameCallback.AddFrameCallback(stageCPtr, Layer.getCPtr(GetRootLayer()));
```

Comment:
seems that frameCallback null check is required...

https://docs.microsoft.com/ko-kr/dotnet/csharp/language-reference/operators/member-access-operators#null-conditional-operators--and-

Reply:
OK